src/main/java/com/coveo/spillway/storage/LimitUsageStorage.java

```diff
@@ -120,6 +120,28 @@
   Map<LimitKey, Integer> debugCurrentLimitCounters();
 
   /**
+   * @param resource The resource for which you want to get the current limit counts
+   * @return A Map of all the enforced limits for a resource and their current count
+   *
+   */
+  Map<LimitKey, Integer> getCurrentLimitCounters(String resource);
```

Comment:
You can easily search by the key's attributes. There's no need to recreate an entire LimitKey to consume what's returned. "getCurrentLimitCount(String resource, String limitName, String property)" could return an Integer as it'll realistically only ever have 1 key, but I preferred consistency.